apps/full-site-editing/full-site-editing-plugin/full-site-editing/templates/class-wp-template-inserter.php

```diff
@@ -0,0 +1,261 @@
+<?php
+/**
+ * Full site editing file.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+/**
+ * Class WP_Template_Inserter
+ */
+class WP_Template_Inserter {
+	/**
+	 * Modern Business header content.
```

Comment:
Let's not mention the theme name anymore! 🙂 (same for the footer variable's comment)

Reply:
Thanks for catching this! Copied the file from the theme and forgot to update the references. :)